PythonRegression/tests/features/steps/blowball_tests_steps.py

```diff
@@ -0,0 +1,89 @@
+from aloe import world, step 
+from util.test_logic import api_test_logic
+from iota import *
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+tests = api_test_logic
+
+config = {}
+
+
+@step(r'find transaction is called with the address "([^"]*)" to find milestones')
+def find_transactions_from_coo(step,address):
```

Comment:
I think this should be in `milestone_steps.py` since it will be used by other features